src/test/java/net/masterthought/cucumber/outputhandlers/FilesystemOutputHandlerTest.java

```diff
@@ -0,0 +1,42 @@
+package net.masterthought.cucumber.outputhandlers;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.util.UUID;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class FilesystemOutputHandlerTest {
+
+    private File outputDirectory;
+
+    @Before
+    public void setup() throws Exception {
+        outputDirectory = new File("target", UUID.randomUUID().toString());
+        Files.createDirectories(outputDirectory.toPath());
+    }
+
+    @After
+    public void cleanup() throws Exception {
+        FileUtils.deleteDirectory(outputDirectory);
+    }
+
+    @Test
+    public void handle_shouldWriteGivenFileContentToGivenFile() throws Exception {
+        final File filesystemOutputHandlerTestOutputFile = new File(outputDirectory, "FilesystemOutputHandler_Testoutput.txt");
+        FilesystemOutputHandler fileSystemOutputHandler = new FilesystemOutputHandler();
+        final String expectedOutput = "This is the FilesystemOutputHandler output";
+        fileSystemOutputHandler.handle(filesystemOutputHandlerTestOutputFile, expectedOutput.getBytes(UTF_8));
+
+        assertThat(filesystemOutputHandlerTestOutputFile).exists();
+
+        String writtenOutput = new String(Files.readAllBytes(filesystemOutputHandlerTestOutputFile.toPath()), UTF_8);
+        assertThat(writtenOutput).isEqualTo(expectedOutput);
+    }
+}
```

Comment:
check if already added comment is not valid for other

can you also update one integration test or even add new one with memory handlers to check whether it works or not?

Reply:
can I test this PR without second or both should be tested together?

Reply:
Added Integration Test

You can Test it without the second PR, they are independend